keyboards/mechwild/sugarglider/ld/STM32F401xE.ld

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
This file is not needed.  F401xE and F411xE support for uf2 with emulated eeprom exists in core, already, and is enabled if the bootloader is properly set.